lib/PicoDb/StatementHandler.php

```diff
@@ -279,12 +279,36 @@ protected function bindParams(PDOStatement $pdoStatement)
         }
 
         foreach ($this->positionalParams as $value) {
-            $pdoStatement->bindValue($i, $value, PDO::PARAM_STR);
+            switch (true) {
+                case is_numeric($value):
```

Comment:
`is_numeric` will return true if:

- Value is a string containing a numeric value
- Value is an int
- Value is a float

Since PDO only has a param type for ints specifically and we want to allow int-only values in actual strings, `is_int` should be used here instead. This means floats and strings containing numeric values are still bound as strings.